### ISSUE TITLE
ReactInputSelection is missing

### ISSUE THREAD

**Issue by maksimu** [schneidmaster/react-bootstrap-maskedinput#3]
When building the app I start getting following error:

```
Error in ./~/react-maskedinput/lib/index.js
Module not found: 'react/lib/ReactInputSelection' in /Users/max/SelectorApp/node_modules/react-maskedinput/lib
```

Not sure what caused this issue. Everything worked until few day ago. Then in the new project when I checked it out form my git and did `npm install && npm run build` I start getting this.

**Comment by maksimu** [comment]
Looks like this issue is caused by the new React, which removed bunch of stuff and the base library that you user are using is broken now. 

See this issue: https://github.com/insin/react-maskedinput/issues/76

**Comment by schneidmaster** [comment]
Yep, as you noted, was caused by an incompatibility with react 15.4 and `react-maskedinput` 3.3.0. Seems to be fixed now with `react-maskedinput` 3.3.1. I don't think any changes are necessary in `react-bootstrap-maskedinput` since we have very permissive version resolution for both dependencies -- there's no way for me to stipulate against that exact broken combination of versions, it's just up to you to bump your installed version if necessary. 😄
